io/plugins/eu.esdihumboldt.hale.io.gml.test/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter2Test.groovy

```diff
@@ -68,6 +67,102 @@ class StreamGmlWriter2Test {
 		TestUtil.startConversionService();
 	}
 
+	/**
+	 * Test if a codespace attribute is automatically added to a GML identifier within an INSPIRE type.
```

Comment:
@Flaminietta  A good additional test would be, to test that the codeSpace is not added automatically if the parent type is not an INSPIRE type.